src/app/personside/infotabs/saksoversikt/DokumentKomponent.tsx

```diff
@@ -39,12 +40,16 @@ interface DispatchProps {
 
 interface StateProps {
     bruker: RestReducer<PersonRespons>;
+    erStandaloneVindu: boolean;
+    valgtDokument?: DokumentMetadata;
+    valgtEnkeltDokument?: Enkeltdokument;
+    visDokument: boolean;
 }
 
 type Props = OwnProps & DispatchProps & StateProps;
 
-const Wrapper = styled.div`
-
+const Wrapper = styled.div<{ valgt: boolean }>`
+  ${props => props.valgt && 'background-color: rgba(0, 0, 0, 0.09);'}
```

Comment:
nice! styled-components 3.0 har en css-"helper" metode som sørger for at css-en lintes ol. Du kan bruke den slik: 
```${props => props.valgt && css`background-color: rgba(0, 0, 0, 0.09);`}```